packages/contracts-bedrock/src/L1/L1CrossDomainMessenger.sol

```diff
@@ -1,8 +1,9 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.15;
+pragma solidity 0.8.25;
 
 // Contracts
 import { CrossDomainMessenger } from "src/universal/CrossDomainMessenger.sol";
```

Comment:
In the past, we did these storage layout spacers, oz v5 initializable uses unstructured storage, much simpler storage layout migration

Reply:
Using oz v5 requires solve ^0.8.20

Reply:
We won't have #12356 in before this unfortunately.